### DISCUSSION TITLE
C files not recognized

### DISCUSSION THREAD

**Discussion by Vardidis** [fetchai/cosmpy#264] [Developers] [answered]
pip installing cosmpy throws errors that .h files cant be found. Using windows 10/ python 3.9.8.
Im not sure whether Im missing some C dependencies or the package is supposed to be run on linux.
Is anyone else facing the same errors?

**Comment by VijaySharma815** [comment]
Error Log:

![image](https://user-images.githubusercontent.com/54273054/193455241-f23bf85d-0cb4-4c77-b93e-f885b5996da5.png)

![image](https://user-images.githubusercontent.com/54273054/193455256-a7744808-ae33-4507-8b87-489a6993152d.png)

**Comment by jrriehl** [comment, marked as answer]
I was able to reproduce this error on Windows. There seems to be a missing install step related to a C compiler. We'll need some further testing on Windows to fix this for future versions.

In the meantime, I managed to get around it by installing Visual Studio Build Tools (https://visualstudio.microsoft.com/downloads/?q=build+tools#build-tools-for-visual-studio-2022 - "Desktop development with C++"). This is quite a bulky install though, so maybe worth uninstalling after installing CosmPy.
